internal/cmd/trust/githubappapprovals/githubappapprovals.go

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package githubappapprovals
+
+import (
+	"github.com/gittuf/gittuf/internal/cmd/common"
+	"github.com/gittuf/gittuf/internal/cmd/trust/persistent"
+	"github.com/gittuf/gittuf/internal/repository"
+	"github.com/spf13/cobra"
+)
+
+type options struct {
+	p       *persistent.Options
+	enable  bool
+	disable bool
+}
+
+func (o *options) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().BoolVar(
+		&o.enable,
+		"enable",
+		false,
+		"mark GitHub app approvals as trusted",
+	)
+
+	cmd.Flags().BoolVar(
+		&o.disable,
+		"disable",
+		false,
+		"mark GitHub app approvals as untrusted",
+	)
+
+	cmd.MarkFlagsOneRequired("enable", "disable") //nolint:errcheck
+	cmd.MarkFlagsMutuallyExclusive("enable", "disable")
+}
+
+func (o *options) Run(cmd *cobra.Command, _ []string) error {
```

Comment:
this one is a deviation from how other CLI commands work. should this be enable and disable subcommands?